playwright/src/main/java/com/microsoft/playwright/impl/BrowserTypeImpl.java

```diff
@@ -105,6 +107,14 @@ private Browser connectImpl(String wsEndpoint, ConnectOptions options) {
     }
   }
 
+  private String getUserAgent() {
+    return String.format("Playwright/%s (%s/%s/%s)",
+      getClass().getPackage().getImplementationVersion(),
```

Comment:
This is java package version which can have a form of `1.15.0-SNAPSHOT` and even the release version is slightly different from the driver version. Do you want to see here driver version or playwright java version?

Reply:
Need to get playwright version i.e `1.15.0`

Reply:
Then this will break in some cases. Also it returns `null` on the to setups that I tried, is it expected?